lib/kmts.rb

```diff
@@ -150,7 +150,9 @@ def log_name(type)
       when :sent
         fname = "kissmetrics#{env}_sent.log"
       when :send
-        fname = Time.now.to_i.to_s + "kissmetrics_#{env}_sending.log"
+        now = Time.now.to_i
```

Comment:
You could retain the milliseconds for some additional entropy: `(Time.now.to_f * 1000).to_i`